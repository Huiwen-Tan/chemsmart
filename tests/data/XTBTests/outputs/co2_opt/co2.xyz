3
co2.xyz    Empirical formula: CO2
O       -1.1970000000    0.0000000000    0.0000000000
O        1.1970000000    0.0000000000    0.0000000000
C        0.0000000000    0.0000000000    0.0000000000
