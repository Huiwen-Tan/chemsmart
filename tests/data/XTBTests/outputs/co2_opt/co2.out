      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.7.1 (edcfbbe) compiled by 'runner@Mac-1732539443871.local' on 2024-11-25

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * C. Bannwarth, E. Caldeweyher, S. Ehlert, A. Hansen, P. Pracht,
     J. Seibert, S. Spicher, S. Grimme, WIREs Comput. Mol. Sci., 2020, 11,
     e01493. DOI: 10.1002/wcms.1493
   
   for GFN2-xTB:
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   for GFN1-xTB:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   for GFN0-xTB:
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   for GFN-FF:
   * S. Spicher and S. Grimme, Angew. Chem. Int. Ed., 2020, 59, 15665-15673.
     DOI: 10.1002/anie.202004239
   
   for ALPB and GBSA implicit solvation:
   * S. Ehlert, M. Stahn, S. Spicher, S. Grimme, J. Chem. Theory Comput.,
     2021, 17, 4250-4261. DOI: 10.1021/acs.jctc.1c00471
   
   for ddCOSMO and CPCM-X implicit solvation:
   * M. Stahn, S. Ehlert, S. Grimme, J. Phys. Chem. A,
     2023, XX, XXXX-XXXX. DOI: 10.1021/acs.jpca.3c04382
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   * E. Caldeweyher, J.-M. Mewes, S. Ehlert and S. Grimme, Phys. Chem. Chem. Phys.
     2020, 22, 8499-8512. DOI: 10.1039/D0CP00502A
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   * J. Koopman and S. Grimme, ACS Omega 2019, 4, 12, 15120-15133.
     DOI: 10.1021/acsomega.9b02011
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   for SPH calculations refer to:
   * S. Spicher and S. Grimme, J. Chem. Theory Comput., 2021, 17, 1701-1714
     DOI: 10.1021/acs.jctc.0c01306
   
   for ONIOM refer to:
   * C. Plett, A. Katbashev, S. Ehlert, S. Grimme, M. Bursch,
     Phys. Chem. Chem. Phys., 2023, 25, 17860-17868. DOI: 10.1039/D3CP02178E
   
   for DIPRO refer to:
   * J. Kohn, N. Gildemeister, S. Grimme, D. Fazzi, A. Hansen,
     J. Chem. Phys., 2023, just accepted.
   
   for PTB refer to:
   * S. Grimme, M. Mueller, A. Hansen, J. Chem. Phys., 2023, 158, 124111.
     DOI: 10.1063/5.0137838
   
   with help from (in alphabetical order)
   P. Atkinson, C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher
   M. Checinski, S. Dohm, S. Ehlert, S. Ehrlich, I. Gerasimov, C. Hölzer
   A. Katbashev, J. Kohn, J. Koopman, C. Lavigne, S. Lehtola, F. März, M. Müller,
   F. Musil, H. Neugebauer, J. Pisarek, C. Plett, P. Pracht, F. Pultar,
   J. Seibert, P. Shushkov, S. Spicher, M. Stahn, M. Steiner, T. Strunk,
   J. Stückrath, T. Rose, and J. Unsleber
   
 * started run on 2025/02/24 at 12:17:14.217     
   ID    Z sym.   atoms
    1    8 O      1, 2
    2    6 C      3

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : /Users/xinglongzhang/miniconda3/envs/chemsmart/bin/xtb /Users/xinglongzhang/chemsmart_test/xtb/co2.xyz --gfn2 --opt vtight --chrg 0 --uhf 0
          coordinate file            : /Users/xinglongzhang/chemsmart_test/xtb/co2.xyz
          omp threads                :                     4


           -------------------------------------------------
          |                 G F N 2 - x T B                 |
           -------------------------------------------------

        Reference                      10.1021/acs.jctc.8b01176
      * Hamiltonian:
        H0-scaling (s, p, d)           1.850000    2.230000    2.230000
        zeta-weighting                 0.500000
      * Dispersion:
        s8                             2.700000
        a1                             0.520000
        a2                             5.000000
        s9                             5.000000
      * Repulsion:
        kExp                           1.500000    1.000000
        rExp                           1.000000
      * Coulomb:
        alpha                          2.000000
        third order                    shell-resolved
        anisotropic                    true
        a3                             3.000000
        a5                             4.000000
        cn-shift                       1.200000
        cn-exp                         4.000000
        max-rad                        5.000000

q/qsh data taken from xtbrestart
CAMM data taken from xtbrestart

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                  12          :
          :  # atomic orbitals                  12          :
          :  # shells                            6          :
          :  # electrons                        16          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                       true          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          :  net charge                          0          :
          :  unpaired electrons                  0          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -10.3748649 -0.103749E+02  0.129E+00    7.06       0.0  T
   2    -10.3772837 -0.241879E-02  0.759E-01    7.40       1.0  T
   3    -10.3780354 -0.751689E-03  0.362E-01    7.28       1.0  T
   4    -10.3780404 -0.493898E-05  0.346E-02    7.28       1.7  T
   5    -10.3780408 -0.415845E-06  0.951E-03    7.28       6.1  T
   6    -10.3780409 -0.103508E-06  0.537E-04    7.28     107.5  T
   7    -10.3780409 -0.122602E-09  0.608E-05    7.28     950.1  T

   *** convergence criteria satisfied after 7 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.8008851             -21.7932
         2        2.0000           -0.7901496             -21.5011
         3        2.0000           -0.6633126             -18.0497
         4        2.0000           -0.6507523             -17.7079
         5        2.0000           -0.6507523             -17.7079
         6        2.0000           -0.6047099             -16.4550
         7        2.0000           -0.5332237             -14.5098
         8        2.0000           -0.5332237             -14.5098 (HOMO)
         9                         -0.2656257              -7.2280 (LUMO)
        10                         -0.2656257              -7.2280
        11                          0.1500787               4.0838
        12                          1.0144364              27.6042
      -------------------------------------------------------------
                  HL-Gap            0.2675980 Eh            7.2817 eV
             Fermi-level           -0.3994247 Eh          -10.8689 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.013 sec
 SCC setup                      ...        0 min,  0.002 sec ( 14.833%)
 Dispersion                     ...        0 min,  0.000 sec (  3.045%)
 classical contributions        ...        0 min,  0.001 sec (  4.402%)
 integral evaluation            ...        0 min,  0.001 sec (  7.870%)
 iterations                     ...        0 min,  0.008 sec ( 59.544%)
 molecular gradient             ...        0 min,  0.001 sec (  5.919%)
 printout                       ...        0 min,  0.001 sec (  4.184%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy             -10.297398913531 Eh    ::
         :: gradient norm              0.145777903348 Eh/a0 ::
         :: HOMO-LUMO gap              7.281711385767 eV    ::
         ::.................................................::
         :: SCC energy               -10.378040872297 Eh    ::
         :: -> isotropic ES            0.032015962077 Eh    ::
         :: -> anisotropic ES          0.003264186896 Eh    ::
         :: -> anisotropic XC          0.001320942382 Eh    ::
         :: -> dispersion             -0.000684457875 Eh    ::
         :: repulsion energy           0.080641958767 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :: total charge               0.000000000000 e     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

      ----------------------------------------------------------- 
     |                   =====================                   |
     |                        A N C O P T                        |
     |                   =====================                   |
     |               Approximate Normal Coordinate               |
     |                Rational Function Optimizer                |
      ----------------------------------------------------------- 
      ----------------------------------------------------------- 
     |                   =====================                   |
     |                        A N C O P T                        |
     |                   =====================                   |
     |               Approximate Normal Coordinate               |
     |                Rational Function Optimizer                |
      ----------------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :   optimization level         verytight          :
          :   max. optcycles                   200          :
          :   ANC micro-cycles                  20          :
          :   degrees of freedom                 4          :
          :.................................................:
          :   RF solver                   davidson          :
          :   write xtbopt.log                true          :
          :   linear (good luck)              true          :
          :   energy convergence         0.1000000E-06 Eh   :
          :   grad. convergence          0.2000000E-03 Eh/α :
          :   maxmium RF displ.          1.0000000          :
          :   Hlow (freq-cutoff)         0.1000000E-01      :
          :   Hmax (freq-cutoff)         5.0000000          :
          :   S6 in model hess.         20.0000000          :
          ...................................................

generating ANC from exact Hessian ...
Using Lindh-Hessian (1995)
 Shifting diagonal of input Hessian by    0.0000000000000000     
 Lowest  eigenvalues of input Hessian
    0.000000    0.000000    0.000000    0.000000    0.000000    0.874927
    0.874927    0.980767    2.874865
 Highest eigenvalues
    0.000000    0.000000    0.874927    0.874927    0.980767    2.874865


........................................................................
.............................. CYCLE    1 ..............................
........................................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -10.3780409 -0.103780E+02  0.146E-06    7.28       0.0  T
   2    -10.3780409 -0.177636E-14  0.873E-07    7.28   66128.9  T
   3    -10.3780409 -0.355271E-14  0.284E-07    7.28  100000.0  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :   -10.2973989 Eh     change       -0.5250911E-11 Eh
   gradient norm :     0.1457779 Eh/α   predicted     0.0000000E+00 (-100.00%)
   displ. norm   :     0.1454904 α      lambda       -0.2120929E-01
   maximum displ.:     0.1454904 α      in ANC's #3, #1, #2, ...

........................................................................
.............................. CYCLE    2 ..............................
........................................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -10.4313354 -0.104313E+02  0.909E-01    8.55       0.0  T
   2    -10.4315979 -0.262474E-03  0.534E-01    8.45       1.0  T
   3    -10.4316068 -0.892964E-05  0.236E-01    8.50       1.0  T
   4    -10.4316341 -0.272568E-04  0.272E-02    8.48       2.1  T
   5    -10.4316341 -0.273534E-07  0.158E-03    8.48      36.4  T
   6    -10.4316341 -0.518822E-08  0.422E-04    8.47     136.9  T
   7    -10.4316341 -0.429168E-11  0.375E-06    8.47   15385.9  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :   -10.3084470 Eh     change       -0.1104813E-01 Eh
   gradient norm :     0.0035993 Eh/α   predicted    -0.1082911E-01 (  -1.98%)
   displ. norm   :     0.0035056 α      lambda       -0.1262036E-04
   maximum displ.:     0.0035056 α      in ANC's #3, #1, #2, ...

........................................................................
.............................. CYCLE    3 ..............................
........................................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -10.4303972 -0.104304E+02  0.231E-02    8.44       0.0  T
   2    -10.4303974 -0.179956E-06  0.136E-02    8.44       4.2  T
   3    -10.4303974 -0.815885E-08  0.619E-03    8.44       9.3  T
   4    -10.4303974 -0.176050E-07  0.710E-04    8.44      81.4  T
   5    -10.4303974 -0.338147E-10  0.181E-05    8.44    3181.6  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :   -10.3084521 Eh     change       -0.5030829E-05 Eh
   gradient norm :     0.0007225 Eh/α   predicted    -0.6308997E-05 (  25.41%)
   displ. norm   :     0.0008791 α      lambda       -0.4217518E-06
   maximum displ.:     0.0008791 α      in ANC's #3, #1, #2, ...

........................................................................
.............................. CYCLE    4 ..............................
........................................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -10.4307078 -0.104307E+02  0.579E-03    8.45       0.0  T
   2    -10.4307078 -0.112634E-07  0.341E-03    8.45      16.9  T
   3    -10.4307078 -0.503521E-09  0.155E-03    8.45      37.3  T
   4    -10.4307078 -0.110010E-08  0.177E-04    8.45     325.6  T
   5    -10.4307078 -0.208189E-11  0.450E-06    8.45   12837.5  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :   -10.3084522 Eh     change       -0.1606178E-06 Eh
   gradient norm :     0.0003575 Eh/α   predicted    -0.2117255E-06 (  31.82%)
   displ. norm   :     0.0002910 α      lambda       -0.6917843E-07
   maximum displ.:     0.0002910 α      in ANC's #3, #2, #1, ...

........................................................................
.............................. CYCLE    5 ..............................
........................................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -10.4306051 -0.104306E+02  0.192E-03    8.45       0.0  T
   2    -10.4306051 -0.123612E-08  0.113E-03    8.45      51.2  T
   3    -10.4306051 -0.555502E-10  0.513E-04    8.45     112.6  T
   4    -10.4306051 -0.120766E-09  0.588E-05    8.45     982.2  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.000 sec
 * total energy  :   -10.3084523 Eh     change       -0.5197417E-07 Eh
   gradient norm :     0.0000003 Eh/α   predicted    -0.3468064E-07 ( -33.27%)
   displ. norm   :     0.0000002 α      lambda        0.1393399E-08
   maximum displ.:     0.0000002 α      in ANC's #3, #2, #1, ...

   *** GEOMETRY OPTIMIZATION CONVERGED AFTER 5 ITERATIONS ***

------------------------------------------------------------------------
 total energy gain   :        -0.0110534 Eh       -6.9361 kcal/mol
 total RMSD          :         0.0823145 a0        0.0436 Å
 total power (kW/mol):        -5.8041268 (step)-4578.7451 (real)
------------------------------------------------------------------------

 ANCopt (total)                0 d,  0 h,  0 min,  0.006 sec
 optimizer setup                ...        0 min,  0.001 sec ( 12.723%)
 model hessian                  ...        0 min,  0.000 sec (  1.481%)
 ANC generation                 ...        0 min,  0.000 sec (  4.304%)
 coordinate transformation      ...        0 min,  0.000 sec (  6.713%)
 single point calculation       ...        0 min,  0.003 sec ( 40.737%)
 optimization log               ...        0 min,  0.000 sec (  3.642%)
 hessian update                 ...        0 min,  0.000 sec (  0.398%)
 rational function              ...        0 min,  0.002 sec ( 26.803%)

================
 final structure:
================
3
 xtb: 6.7.1 (edcfbbe)
O           -1.14365140482700        0.00000000000000        0.00000000000000
O            1.14365140482700        0.00000000000000        0.00000000000000
C           -0.00000000000000       -0.00000000000000       -0.00000000000000

 Bond Distances (Angstroems)
 ---------------------------
O1-C3=1.1437         O2-C3=1.1437         C3-O1=1.1437         C3-O2=1.1437         
 O  C  Rav=1.1437 sigma=0.0000  Rmin=1.1437  Rmax=1.1437     2

 selected bond angles (degree)
 --------------------
O2-C3-O1=180.00                

 selected dihedral angles (degree)
 ---------------------------------

           -------------------------------------------------
          |                Final Singlepoint                |
           -------------------------------------------------

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                  12          :
          :  # atomic orbitals                  12          :
          :  # shells                            6          :
          :  # electrons                        16          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          :  net charge                          0          :
          :  unpaired electrons                  0          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -10.4306051 -0.104306E+02  0.150E-06    8.45       0.0  T
   2    -10.4306051  0.177636E-14  0.870E-07    8.45   66388.8  T
   3    -10.4306051 -0.177636E-14  0.604E-08    8.45  100000.0  T

   *** convergence criteria satisfied after 3 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.8108159             -22.0634
         2        2.0000           -0.7966848             -21.6789
         3        2.0000           -0.6648060             -18.0903
         4        2.0000           -0.6634711             -18.0540
         5        2.0000           -0.6634711             -18.0540
         6        2.0000           -0.5998338             -16.3223
         7        2.0000           -0.5344396             -14.5428
         8        2.0000           -0.5344396             -14.5428 (HOMO)
         9                         -0.2239572              -6.0942 (LUMO)
        10                         -0.2239572              -6.0942
        11                          0.3098931               8.4326
        12                          1.3642451              37.1230
      -------------------------------------------------------------
                  HL-Gap            0.3104824 Eh            8.4487 eV
             Fermi-level           -0.3791984 Eh          -10.3185 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.000 sec
 SCC setup                      ...        0 min,  0.000 sec (  6.900%)
 Dispersion                     ...        0 min,  0.000 sec (  1.493%)
 classical contributions        ...        0 min,  0.000 sec (  1.751%)
 integral evaluation            ...        0 min,  0.000 sec ( 15.345%)
 iterations                     ...        0 min,  0.000 sec ( 34.552%)
 molecular gradient             ...        0 min,  0.000 sec ( 30.896%)
 printout                       ...        0 min,  0.000 sec (  7.775%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy             -10.308452289174 Eh    ::
         :: gradient norm              0.000000274756 Eh/a0 ::
         :: HOMO-LUMO gap              8.448655876193 eV    ::
         ::.................................................::
         :: SCC energy               -10.430605117256 Eh    ::
         :: -> isotropic ES            0.032324567806 Eh    ::
         :: -> anisotropic ES          0.003405663023 Eh    ::
         :: -> anisotropic XC          0.000432280404 Eh    ::
         :: -> dispersion             -0.000687152300 Eh    ::
         :: repulsion energy           0.122152828081 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :: total charge              -0.000000000000 e     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.8108159             -22.0634
         2        2.0000           -0.7966848             -21.6789
         3        2.0000           -0.6648060             -18.0903
         4        2.0000           -0.6634711             -18.0540
         5        2.0000           -0.6634711             -18.0540
         6        2.0000           -0.5998338             -16.3223
         7        2.0000           -0.5344396             -14.5428
         8        2.0000           -0.5344396             -14.5428 (HOMO)
         9                         -0.2239572              -6.0942 (LUMO)
        10                         -0.2239572              -6.0942
        11                          0.3098931               8.4326
        12                          1.3642451              37.1230
      -------------------------------------------------------------
                  HL-Gap            0.3104824 Eh            8.4487 eV
             Fermi-level           -0.3791984 Eh          -10.3185 eV

     #   Z          covCN         q      C6AA      α(0)
     1   8 O        0.856    -0.232    18.599     5.782
     2   8 O        0.856    -0.232    18.599     5.782
     3   6 C        1.711     0.464    21.600     7.524

 Mol. C6AA /au·bohr⁶  :        174.800200
 Mol. C8AA /au·bohr⁸  :       4029.884814
 Mol. α(0) /au        :         19.088396


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom

 ---------------------------------------------------------------------------
     #   Z sym  total        # sym  WBO       # sym  WBO       # sym  WBO
 ---------------------------------------------------------------------------
     1   8 O    2.370 --     3 C    1.968     2 O    0.402
     2   8 O    2.370 --     3 C    1.968     1 O    0.402
     3   6 C    3.936 --     2 O    1.968     1 O    1.968
 ---------------------------------------------------------------------------

Topologies differ in total number of bonds
Writing topology from bond orders to xtbtopo.mol

molecular dipole:
                 x           y           z       tot (Debye)
 q only:        0.000      -0.000      -0.000
   full:        0.000       0.000      -0.000       0.000
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:       -2.169       0.000       1.084       0.000       0.000       1.084
  q+dip:       -3.107       0.000       1.553       0.000      -0.000       1.553
   full:       -4.360      -0.000       2.180       0.000      -0.000       2.180

           -------------------------------------------------
          |                Geometry Summary                 |
           -------------------------------------------------

      molecular mass/u    :       44.0095457
   center of mass at/Å    :       -0.0000000       0.0000000       0.0000000
  moments of inertia/u·Å² :        0.1397148E-15   0.4185248E+02   0.4185248E+02
rotational constants/cm⁻¹ :        0.1206575E+18   0.4027869E+00   0.4027869E+00

 * 2 selected distances

     #   Z          #   Z                                           value/Å
     1   8 O        3   6 C                                       1.1436515 (max)
     2   8 O        3   6 C                                       1.1436515

 * 1 distinct bonds (by element types)

   Z      Z             #   av. dist./Å        max./Å        min./Å
   6 C    8 O           2     1.1436515     1.1436515     1.1436515


optimized geometry written to: xtbopt.xyz


           -------------------------------------------------
          | TOTAL ENERGY              -10.308452289174 Eh   |
          | GRADIENT NORM               0.000000274756 Eh/α |
          | HOMO-LUMO GAP               8.448655876193 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2025/02/24 at 12:17:14.309     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.089 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.250 sec
 * ratio c/w:     2.811 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.013 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.040 sec
 * ratio c/w:     3.114 speedup
 ANC optimizer:
 * wall-time:     0 d,  0 h,  0 min,  0.016 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.053 sec
 * ratio c/w:     3.333 speedup

